django/db/backends/oracle/creation.py

```diff
@@ -75,8 +75,12 @@ def _create_test_db(self, verbosity=1, autoclobber=False, keepdb=False):
             if verbosity >= 1:
                 print("Creating test user...")
             try:
-                self._create_test_user(cursor, parameters, verbosity)
+                self._create_test_user(cursor, parameters, verbosity, keepdb)
             except Exception as e:
+                # if we want to keep the db then we want also to keep user,
```

Comment:
I see you copy/pasted but I think "If we want to keep the db, then we also want to keep the user." is sufficient for this comment.